lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -0,0 +1,1387 @@
+# frozen_string_literal: true
+
+require "dry-inflector"
```

Comment:
```suggestion
```

`Dry::Inflector` isn't referenced in the code. It's a dependency that the consumer of this class must provide.